doc/graphviz/cache_fetch.dot

```diff
@@ -39,16 +39,18 @@ digraph cache_fetch {
 		/* vbf_stp_startfetch() */
 		v_b_f [
 			    shape=record
-			    label="{vbf_stp_startfetch:|{vcl_backend_fetch\{\}|bereq.*}|{fail|abandon|<fetch>fetch}}"
+			    label="{vbf_stp_startfetch:|{vcl_backend_fetch\{\}|bereq.*}|{<error>error|fail|abandon|<fetch>fetch}}"
 		]
+		v_b_f:error:s -> v_b_e
 		v_b_f:fetch:s -> v_b_hdrs [style=bold]
 		v_b_hdrs [ label="send bereq,\nread beresp (headers)"]
 		v_b_hdrs -> v_b_r [style=bold]
 		v_b_hdrs -> v_b_e
 		v_b_r [
 			    shape=record
-			    label="{vbf_stp_startfetch:|{vcl_backend_response\{\}|{bereq.*|beresp.*}}|{fail|{retry|{<max>max?|<retry>ok?}}|abandon|{deliver or pass|{<fetch_304>304?|<non_304>other?}}}}"
+			    label="{vbf_stp_startfetch:|{vcl_backend_response\{\}|{bereq.*|beresp.*}}|{<error>error|{retry|{<max>max?|<retry>ok?}}|abandon|{deliver or pass|{<fetch_304>304?|<non_304>other?}}}}"
```

Comment:
Should `return (fail)` disappear from here?

Reply:
No, because breaking changes are bad, and fail was made as a standard return for everywhere so it should definitely stay.

Reply:
Whoops, fixed in latest force push